.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: elixir
 sudo: false
-elixir: '1.5'
-otp_release: '20.0'
+elixir:
+  - 1.6.4
+otp_release:
+  - 19.3
+  - 20.3
```

Comment:
mixing real changes with formatting changes is dangerous but it's ok here I think :)